### ISSUE TITLE
SimpleMenu not support for dark mode

### ISSUE THREAD

**Issue by tcqq** [Gericop/Android-Support-Preference-V7-Fix#152]
@Gericop I changed 
```
    <style name="Theme.MyTheme" parent="@style/PreferenceFixTheme.Light.NoActionBar">
    </style>
```
to 
```
    <style name="Theme.MyTheme" parent="@style/PreferenceFixTheme.NoActionBar">
    </style>
```
But the background color of "SimpleMenu" has not changed.

![image](https://user-images.githubusercontent.com/10644057/39076695-85e70c3c-452f-11e8-8860-441e1e6d2e9c.png)

**Comment by Gericop** [comment]
Will be fixed in the next release.

**Comment by Gericop** [comment]
Fixed in v**27.1.1.1**.

**Comment by tcqq** [comment]
Thanks:)
